web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -64,26 +65,34 @@ public class RouterServerTest extends ReplicaGraphHopperTest {
             Timestamp.newBuilder().setSeconds(Instant.parse("2019-10-13T13:30:00Z").toEpochMilli() / 1000).build();
     private static final double[] REQUEST_ORIGIN_1 = {38.74891667931467,-121.29023848101498}; // Roseville area
     private static final double[] REQUEST_ORIGIN_2 = {38.59337420024281,-121.48746937746185}; // Sacramento area
-    private static final double[] REQUEST_DESTINATION = {38.55518457319914,-121.43714698730038}; // Sacramento area
+    private static final double[] REQUEST_ORIGIN_3 = {38.508810062393245,-121.5085223084316}; // South of Sacramento area
+    private static final double[] REQUEST_DESTINATION_1 = {38.55518457319914,-121.43714698730038}; // Sacramento area
+    private static final double[] REQUEST_DESTINATION_2 = {38.64478548196401,-121.34168802760543}; // North of Sacramento area
+
     // PT_REQUEST_1 should force a transfer between routes from 2 distinct feeds
-    private static final RouterOuterClass.PtRouteRequest PT_REQUEST_1 = createPtRequest(REQUEST_ORIGIN_1, REQUEST_DESTINATION);
+    private static final RouterOuterClass.PtRouteRequest PT_REQUEST_1 = createPtRequest(REQUEST_ORIGIN_1, REQUEST_DESTINATION_1);
     // PT_REQUEST_2 should force a transfer between routes from the same feed
-    private static final RouterOuterClass.PtRouteRequest PT_REQUEST_2 = createPtRequest(REQUEST_ORIGIN_2, REQUEST_DESTINATION);
+    private static final RouterOuterClass.PtRouteRequest PT_REQUEST_2 = createPtRequest(REQUEST_ORIGIN_2, REQUEST_DESTINATION_1);
+    // PT_REQUEST_3 tests park-and-ride routing, with custom access/egress modes
+    private static final RouterOuterClass.PtRouteRequest PT_REQUEST_3  = createPtRequest(REQUEST_ORIGIN_1, REQUEST_DESTINATION_1, "car", "foot");
```

Comment:
nit: maybe we can give these new requests (and maybe even the dests) more descriptive names, like `PT_REQUEST_PARK_AND_RIDE` and `PT_REQUEST_PARK_AND_RIDE_WITH_TRANSFER`?

Reply:
heh my long comments weren't insightful enough, eh? 😅 

Great suggestion, updated all the request names here to something much more readable. Decided to hold off on the destinations for now because they're pretty random and I don't think I want to highlight the exact locations in unit test, more what request "situation" the test is exercising (which the request name encapsulates now)